### DISCUSSION TITLE
avoid having to start from scratch with langchain when the project becomes more custom: support code export

### DISCUSSION THREAD

**Discussion by rmminusrslash** [FlowiseAI/Flowise#553] [Ideas]
I love flowise for quick prototyping. Great tool! As a developer, I sometimes run into the limits of the tool and would like to drop down to the level of the code to make further changes. Currently, I have to start with langchain from scratch if this happens in the lifecycle of a project.

I am aware this might be out of scope since you focus on low-code, but ask in case you find it interesting:

Do you think it would make sense to make the final created code available in python so there is less risk if the project turns out to be more complex than expected?

**Comment by chungyau97** [comment]
Hi @rmminusrslash,

Code export feature based on your created flow is on the roadmap but it's not the highest priority based on current schedule.

**Comment by rmminusrslash** [comment]
Hi @chungyau97,

that makes sense, thank you for the feedback. I will give the tool another try when it's there.
